docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -52,7 +52,7 @@
     * [Jedis](https://github.com/xetorthio/jedis) 2.x
     * [Redisson](https://github.com/redisson/redisson) Easy Java Redis client 3.5.2+
     * [Lettuce](https://github.com/lettuce-io/lettuce-core) 5.x
-  * [MongoDB Java Driver](https://github.com/mongodb/mongo-java-driver) 2.13-2.14, 3.4.0-3.5.0, 3.7.x-3.11.1
+  * [MongoDB Java Driver](https://github.com/mongodb/mongo-java-driver) 2.13-2.14, 3.4.0-3.5.0, 3.6.0-3.6.4, 3.7.x-3.11.1
```

Comment:
I think this should be 3.4 - 3.11 ?

Reply:
Thanks for your reviewed,I have resubmitted it.